modules/tinymce/src/plugins/bbcode/main/ts/Plugin.ts

```diff
@@ -10,6 +10,7 @@ import PluginManager from 'tinymce/core/api/PluginManager';
 import * as Convert from './core/Convert';
 
 export default () => {
+  console.warn('Notice: The bbcode plugin has been marked for removal from TinyMCE 6.0');
```

Comment:
Just gonna comment on one of these
```suggestion
  console.warn('Notice: The bbcode plugin has been deprecated and will be removed in TinyMCE 6.0');
```

Reply:
I also don't think we need the `Notice` in this if we're logging it as a warning. I know the JIRA said to use this string but I think it's wrong honestly...

Reply:
This is also the wrong place to be putting the warning... it should only be logged if actually being used, so this should be instead the `PluginManager.add()` logic, not outside of it.

Reply:
I agree with removing the notice, but I believe "marked for removal" is still technically correct so need some guidance here

Reply:
```suggestion
  console.warn('The bbcode plugin has been deprecated and marked for removal in TinyMCE 6.0');
```